src/modules/vault/components/SignersDetails.tsx

```diff
@@ -144,7 +144,8 @@ const SignersDetails = ({ vault, ...rest }: ISignersDetailsExtendedProps) => {
           Signers
         </Text>
         <Badge p={0} rounded="lg" px={3} fontWeight="medium" variant="gray">
-          Required signers {vault?.data?.minSigners}/
+          Required signers{' '}
+          {JSON.parse(vault.data.configurable).SIGNATURES_COUNT}/
```

Comment:
add parse on query of vault data, and type

Reply:
Done!